tests/integration/api/controllers/login.spec.js

```diff
@@ -157,6 +162,31 @@ describe('login', () => {
         .then(() => loginWithData({ user: user.username, password }))
         .then(response => expectRedirectToPasswordReset(response));
     });
+    
+
+    it('should fail if sso user', async () => {
+      await setupOidcLoginSettings('clientId');
+
+      const newUser = {
+        username: user.username,
+        oidc: true,
+        place: {
+          type: 'health_center',
+          name: 'Online place',
+          parent: 'PARENT_PLACE'
+        },
+        contact: {
+          name: 'OnlineUser'
+        },
+        roles: ['district_admin'], 
+      };
```

Comment:
suggestion: none of these other properties matter, right?  (Can just be the defaults from `user`).

```suggestion
      const newUser = {
        ...user,
        oidc: true,
      };
```